src/exp_log_geo.jl

```diff
@@ -17,11 +17,15 @@ function exp(M::AbstractManifold, p, X)
     exp!(M, q, p, X)
     return q
 end
-exp(M::AbstractManifold, p, X, t::Real) = exp(M, p, t * X)
+function exp(M::AbstractManifold, p, X, t::Number)
```

Comment:
We could actually implement both `exp`s also as

```
function exp(M::AbstractManifold, p, X, t::Number=one(eltype(X)))
```

and even advice this generally – or would that kill performance?

Reply:
I'm not sure actually, what would be the benefit here?

Reply:
I am also not 100% sure but it would make this change a bit easier for cases with or without t? I you do not feel that this is useful we can also leave that out.

Reply:
Well, the current variant definitely works and I don't see how this change would make anything significantly easier.